.eslintrc.js

```diff
@@ -3,16 +3,31 @@ module.exports = {
         "browser": true,
         "es6": true
     },
-    "extends": "eslint:recommended",
+    "extends": [
+        "eslint:recommended",
+        // "plugin:@typescript-eslint/recommended",
+        // "plugin:@typescript-eslint/recommended-requiring-type-checking"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint"],
     "parserOptions": {
         "ecmaVersion": 2020,
-        "sourceType": "module"
+        "sourceType": "module",
+        "project": "./tsconfig.json"
     },
     "rules": {
+        "semi": "error",
+        "quotes": "error",
         "no-console": "off",
         "no-empty": "off",
         "no-prototype-builtins": "off",
-        "no-unused-vars": "warn"
+        "no-unused-vars": "warn",
+        "quote-props": ["warn", "consistent"],
+        "no-multi-spaces": ["warn", { "ignoreEOLComments": true }],
+        "@typescript-eslint/no-unused-vars": "warn"
+        // "@typescript-eslint/no-misused-promises": "error",
+        // "@typescript-eslint/no-floating-promises": 2,
+        // "@typescript-eslint/explicit-function-return-type": "error"
```

Comment:
Rather than taking the to resolve the warnings, I opted to disable this rules for now.

Reply:
time